terraso_backend/apps/core/models/users.py

```diff
@@ -47,6 +47,7 @@ class User(AbstractUser):
 
     username = None
     email = models.EmailField(unique=True)
+    profile_image = models.URLField(null=True)
```

Comment:
It's a detail, but to avoid having two values for "no value" (null and empty string), it's a common practice to define the field this way: `models.URLField(blank=True, default="")`. It means the model allows no value for this field and the default value (when nothing is informed) is going to be and empty string. This is how `Group.website` and `Landscape.website` is being declared.

Remember to generate the migration again after the change.